lib/instrumentation/modules/http2.js

```diff
@@ -5,6 +5,7 @@ var eos = require('end-of-stream')
 var shimmer = require('../shimmer')
 var symbols = require('../../symbols')
 var { parseUrl } = require('../../parsers')
+var { getDestination } = require('../context')
```

Comment:
Not sure if its best to keep it on `context` file. I am not sure we follow any convention on the Node.js agent. I am open to suggestions.